### PR TITLE
[14.0] fix product state

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**

### CODE REVIEW

File: product_state/views/product_template.xml [hunk 1]
```diff
@@ -144,7 +144,7 @@
         <field name="priority">50</field>
         <field name="arch" type="xml">
             <field name="type" position="after">
-                <field name="state" />
+                <field name="product_state_id" />
```
Comment (sebalix) [inline product_state/views/product_template.xml:147]: Why this one instead of the other?

**Comment by rousseldenis** [comment]
@xavier-bouquiaux Could you explain in description, both modifications ?

**Comment by rousseldenis** [comment]
@xavier-bouquiaux

File: product_state/views/product_template.xml (hunk 1 above)
Comment (xavier-bouquiaux) [reply product_state/views/product_template.xml:147]: state is just a compute of product_state_id.code <br/>
You can use more odoo function by showing directly the many2one

**Pull request closed without merging**